sonar-ws-generated/src/main/protobuf/ws-qualitygates.proto

```diff
@@ -118,5 +118,36 @@ message UpdateConditionWsResponse {
   optional int32 period = 6;
 }
 
+// GET api/qualitygates/show
```

Comment:
Just a reminder here : can we discuss how we are using Protobuf, is there any specific usage ?

Reply:
The usage of protobuf allows to quickly write response of WS. 
As you have may seen in my PR, returning JSON is a pain, and not easily testable.